index.js

```diff
@@ -24,7 +45,7 @@ class PQueue {
 			this._resolveEmpty();
 		}
 	}
-	add(fn) {
+	add(fn, options) {
```

Comment:
Probably options is better, not to limit the parameters any Queue class one might implement.